### PR TITLE
build(cli): env show ask

### PR DESCRIPTION
**Pull request by huanjani**
Added env show ask method and unit tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by huanjani** [comment]
example manual output (there is only one env in each of these projects):
```
$ ecs-preview env show

? Which project's environments would you like to show?  [Use arrows to move, type to filter, ? for more help]
> dinder
  e2e-init-1583862289

$
```

File: internal/pkg/cli/env_show.go [hunk 1]
```diff
@@ -53,14 +67,96 @@ func (o *showEnvOpts) Validate() error {
...
+		return nil
+	}
+	projNames, err := o.retrieveProjects()
+	if err != nil {
+		return err
+	}
+	if len(projNames) == 0 {
+		return fmt.Errorf("no project found: run %s please", color.HighlightCode("project init"))
+	}
+	proj, err := o.prompt.SelectOne(
```
[the remark this answers is not in the document]
Comment (huanjani) [reply internal/pkg/cli/env_show.go:92]: Yes! Thanks!

File: internal/pkg/cli/env_show.go [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 	"github.com/spf13/cobra"
 )
 
+const (
+	environmentShowProjectNamePrompt     = "Which project's environments would you like to show?"
+	environmentShowProjectNameHelpPrompt = "A project groups all of your applications together."
+	environmentShowEnvNamePrompt         = "Which environment of %s would you like to show?"
```
Comment (efekarakus) [inline internal/pkg/cli/env_show.go:19]: nit: We usually prefix string constants that contain format verbs like "%s" with `fmt`.   
So something like: `fmtEnvShowNamePrompt` this makes it easy to notice that the string itself shouldn't be printed as is but needs some data.

File: internal/pkg/cli/env_show.go [hunk 3]
```diff
@@ -53,24 +66,110 @@ func (o *showEnvOpts) Validate() error {
...
+func (o *showEnvOpts) askProject() error {
+	if o.ProjectName() != "" {
+		return nil
+	}
+	projNames, err := o.retrieveProjects()
+	if err != nil {
+		return err
+	}
+	if len(projNames) == 0 {
+		return fmt.Errorf("no project found: run %s please", color.HighlightCode("project init"))
```
Comment (efekarakus) [inline internal/pkg/cli/env_show.go:89]: very nice!

File: internal/pkg/cli/env_show.go (hunk 2 above)
Comment (huanjani) [reply internal/pkg/cli/env_show.go]: Thanks! Made the change here and in test file.

**Pull request merged**